### ISSUE TITLE
Minor PPA improvement

### ISSUE THREAD

**Issue by dpetrisko** [bespoke-silicon-group/bsg_manycore#507]
https://github.com/bespoke-silicon-group/bsg_manycore/blob/master/v/vanilla_bean/load_packer.v#L50

Could be made a one-hot mux, since an op will only have 1 size at a time

**Comment by drichmond** [comment]
Sounds like a task for @LaurenChoquer

**Comment by taylor-bsg** [comment]
I think rewriting the entire module so that bit-by-bit (or groups of bits)
we define the output could pay some dividends. Modifying the if then else
statement along I am less certain.  Probably worth looking at what
synthesis is doing first though.

M

**Comment by dpetrisko** [comment]
Yeah we saw an improvement switching to one-hot for a very similar path bp. If we get a better result with bit muxing, will definitely implement it there to (this is the current critical path)

**Comment by taylor-bsg** [comment]
Oh maybe share the link to the refactored module for Lauren

**Comment by dpetrisko** [comment]
https://github.com/black-parrot/black-parrot/blob/master/bp_be/src/v/bp_be_dcache/bp_be_dcache.sv#L625
